vignettes/polars.Rmd

```diff
@@ -1,8 +1,8 @@
 ---
-title: "Introduction to rpolars"
+title: "Introduction to polars in R"
 output: rmarkdown::html_vignette
 vignette: >
-  %\VignetteIndexEntry{Introduction to rpolars}
+  %\VignetteIndexEntry{Introduction to polars in R}
```

Comment:
Grammatically, "polars _in_ R" sounds a bit off. Maybe change to "An Introduction to Polars from R" (or just "Polars from R").